src/common/syncjournaldb.cpp

```diff
@@ -950,6 +950,7 @@ bool SyncJournalDb::setFileRecord(const SyncJournalFileRecord &_record)
     }
 }
 
+// TODO: filename -> QBytearray?
```

Comment:
Why?

Reply:
Looks like its basically always used as a byte array and converted back and forth but it would have bloated this review....